synthesizer/program/src/logic/finalize_operation/bytes.rs

```diff
@@ -134,17 +134,13 @@ impl<N: Network> ToBytes for FinalizeOperation<N> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use console::network::Testnet3;
-
-    type CurrentNetwork = Testnet3;
 
     #[test]
     fn test_bytes() {
         for expected in crate::logic::finalize_operation::test_helpers::sample_finalize_operations() {
             // Check the byte representation.
             let expected_bytes = expected.to_bytes_le().unwrap();
             assert_eq!(expected, FinalizeOperation::read_le(&expected_bytes[..]).unwrap());
-            assert!(FinalizeOperation::<CurrentNetwork>::read_le(&expected_bytes[1..]).is_err());
```

Comment:
This will always be fixed size. You can keep it.

Reply:
It may be fixed size for each variant individually, but the variants have different lengths and this test has been flaky in the past due to this assertion.